Update index.md (#2688)
umbraco/UmbracoDocs: Umbraco-Cloud/Security/index.md

For more information see [the related GitHub issue](https://github.com/Azure/app-service-announcements/issues/12).

> ## Blacklist specific IPs from accessing you website

Reviewer: As we prefer not to use the word "blacklist", I've tried to rewrite this heading.

```suggestion
## Website access based on specific IPs
```

Author: Maybe "Deny specific IPs from accessing your website" instead?

Reviewer: Yes, that's also a good option! 👍

> You can block people and bots(e.g. a malicious scanning bot) from accessing you website by blacklisting their IPs.

Reviewer: ```suggestion
You can block people and bots(e.g. a malicious scanning bot) from accessing your website by adding their IP addresses to a deny-list.
```

The following rule can be added to your web.config file in the `system.webServer/rewrite/rules/` section.

```xml
<rule name="RequestBlockByIP" patternSyntax="Wildcard" stopProcessing="true">
    <match url="*"/>
    <conditions>
    <add input="{REMOTE_ADDR}" negate="false" pattern="123.123.123.123"/>
    </conditions>
    <action type="AbortRequest"/>
</rule>
```

> This will result with anyone using the 123.123.123.123 IP getting a 502 error but you can, of course, choose you own error.

Reviewer: ```suggestion
For anyone using the 123.123.123.123 IP, this will result in them getting a 502 error. You can, of course, choose your own error.
```
